src/main/java/org/opensearch/flowframework/indices/FlowFrameworkIndicesHandler.java

```diff
@@ -468,7 +472,8 @@ public void updateFlowFrameworkSystemIndexDocWithScript(
                 // TODO: Also add ability to change other fields at the same time when adding detailed provision progress
                 updateRequest.script(script);
                 updateRequest.setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE);
-                // TODO: decide what condition can be considered as an update conflict and add retry strategy
+                updateRequest.retryOnConflict(5);
```

Comment:
How are we deciding the numbers here `5` here and `3` on L441?

Reply:
3 was used across the board in ml-commons. Had 5 here cause the script update was the one that happens more often, can change it back to 3

Reply:
check that it works well with 3 also, moved both 3 to keep consistent for now